cpp/test/sparse/neighbors/connect_components.cu

```diff
@@ -117,8 +117,15 @@ class ConnectComponentsTest
      * 3. connect_components to fix connectivities
      */
     raft::linkage::FixConnectivitiesRedOp<value_idx, value_t> red_op(colors.data(), params.n_row);
-    raft::linkage::connect_components<value_idx, value_t>(
-      handle, out_edges, data.data(), colors.data(), params.n_row, params.n_col, red_op);
+    raft::linkage::connect_components<value_idx, value_t>(handle,
+                                                          out_edges,
+                                                          data.data(),
+                                                          colors.data(),
+                                                          params.n_row,
+                                                          params.n_col,
+                                                          red_op,
+                                                          params.n_row,
+                                                          params.n_col);
```

Comment:
Can we test with a batch size that is smaller than the number of rows and the number of columns? Just to make sure the batching logic is excercised.